packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -657,10 +663,66 @@ function makeAugmentedSchema(
                 {}
             );
 
+            const fulltextResultDescription = `The result of a fulltext search on an index of ${node.name}`;
+            const fulltextWhereDescription = `The input for filtering a fulltext query on an index of ${node.name}`;
+            const fulltextSortDescription = `The input for sorting a fulltext query on an index of ${node.name}`;
+
             composer.createInputTC({
                 name: `${node.name}Fulltext`,
                 fields,
             });
+
+            if (!fulltextScoreWhereCreated) {
```

Comment:
**Non-blocking**
**If possible and easy** Consider moving the fulltext types to a separate function, to avoid increasing the size of `makeAugmentedSchema`

Reply:
Yeah I'll do this. I did actually consider it because I also think it'd be nicer but seeing as it hadn't been done anywhere else I wanted to avoid stylistic changes